contracts/src/PDPService.sol

```diff
@@ -325,6 +343,8 @@ contract PDPService {
 
         // Set the next challenge epoch.
         nextChallengeEpoch[setId] = block.number + challengeFinality; 
+        bytes memory extraData = abi.encode(seed);
```

Comment:
From our discussions about single owner versus mixed proof sets, it's important for the event observer to know the total size of the proof set when it's proven. Are you relying on them tracking this from the sequence of adds and removes, including of data that is not of interest? Perhaps the convenience of including the total size with the proof event is worth the overhead.

Reply:
> Are you relying on them tracking this from the sequence of adds and removes, including of data that is not of interest?

Yes I think this is essentially required either after the fact or with a PDP application using logic to guard what goes into the proof set.  But I think this is fine metadata to include here too.